crates/core/src/factories/mem_op_store.rs

```diff
@@ -234,6 +234,55 @@ impl OpStore for Kitsune2MemoryOpStore {
         })
     }
 
+    fn retrieve_op_ids_bounded(
```

Comment:
This isn't particularly efficient, I'm hoping it's clear though and for tests I'm not expecting us to be dealing with more than a few hundred ops, maybe a few thousand.